conteudo/grau-de-antipoda-prop.tex

```diff
@@ -0,0 +1,29 @@
+\subsection{Grau de antípoda} %afirmação aqui significa teorema/proposição/colorário/lema
+\label{grau-de-antipoda-prop}
+\begin{titlemize}{Lista de dependências}
+    \item \hyperref[homologia-singular-def]{Homologia singular};\\
+    \item \hyperref[homomorfismo-de-homologias-singulares-induzido-prop]{Homomorfismo de homologias singulares induzido};\\
+    \item \hyperref[homologia-singular-de-S1-prop]{Homologia singular da circunferência};\\
+    \item \hyperref[grupo-de-homologia-singular-de-n-esfera-prop]{Grupo de homologia singular de n-esfera};\\
+    \item \hyperref[grau-de-funcoes-em-esferas-def]{Grau de funções em esferas};\\
+    \item \hyperref[propriedades-de-grau-de-funções-prop]{Propriedade de grau de funções em esferas};\\
+    \item \hyperref[grau-da-reflexao-prop]{Grau da reflexão}
+\end{titlemize}
+
+\begin{defi}
+    Uma \textbf{antípoda} é uma função de $a:\mathbb{S}^n\rightarrow \mathbb{S}^n$ dada por $a(x)=-x$.
+\end{defi}
+
+\begin{lemma}
+    A antípoda da esfera $\mathbb{S}^n$ tem grau $(-1)^{n+1}$
+\end{lemma}
+
+\begin{dem}
+    Para cada $1\le i\le n+1$, seja $r_i: \mathbb{S}^n\rightarrow \mathbb{S}^n$ a reflexão da i-ésima coordenada. A antípoda fatora-se como a composição $a=r_1\circ ...\circ r_{n+1}$. Segue das propriedades de grau de funções em esferas que 
+    \[deg(a)=deg(r_1)\cdot...\cdot deg(r_{n+1})=(-1)^{n+1}\]
+\end{dem}
+
+\begin{titlemize}{Lista de consequências}
```

Comment:
pode tirar essa parte se não for usar.

Reply:
Acho que eu vou adcionar teorema de bola cabeluda na consequência.